aws-marketplace-install.md

```diff
@@ -1,13 +1,15 @@
 AWS Marketplace Install
 =======================
 
-This document provides the steps for installing the Kubecost Enterprise product from the AWS marketplace. [More info on different tiers.](https://kubecost.com/pricing)
```

Comment:
This is tricky because the user actually needs a product key before they can fully access all Enterprise functionality. I guess we could look at changing this.

Reply:
@dwbrown2 Hi Webb, for your information, I commit new changes to update Helm command with Prometheus image on AWS MP ECR repo. This is a first update to reflect the new version added for current AWS MP listing. In this update, I use the general term "Kubecost from AWS Marketplace" instead. We can revisit and update the doc in the next update as well if it is needed. Thanks